api/src/Mail/Account.php

```diff
@@ -397,7 +397,7 @@ public function imapServer($_adminConnection=false, $_timeout=null)
 	 */
 	public function is_imap($try_connect=true)
 	{
-		if (empty($this->acc_imap_host) || empty($this->acc_imap_username) || empty($this->acc_imap_password))
+		if (empty($this->acc_imap_host) && empty($this->acc_imap_username) && empty($this->acc_imap_password))
 		{
```

Comment:
That goes to far, empty host makes not sense!
```
if (empty($this->acc_imap_host) || empty($this->acc_imap_username) && empty($this->acc_imap_password))
```
Thought these kind of changes tend to break working installations ...